CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ### Enhancements
 * <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
-* None.
+* Fulltext search index can now be defined on ObjectStore Property (PR [#6100](https://github.com/realm/realm-core/pull/6100))
```

Comment:
Please add explanation of what this feature will do.

Reply:
I have added explanation in this PR. I think this is what the link is for.

Reply:
Do you think this PR describes in sufficient details the API changes that the SDKs need to expose the new functionality? Both Featurewise and functionality wise it's my impression that a more detailed description would help SDK teams adopt this much easier without having to review the details in this PR.